conf/modules/assembly_variant_calling.config

```diff
@@ -13,6 +13,10 @@
 process {
 
     withName: DIPCALL {
```

Comment:
Generally, for naming the module we use `.*SUBWORKFLOW_NAME:MODULE_NAME`

Reply:
Is it ok if I change all modules later (created an issue #36), or do you think I should start with this one now?

Reply:
It's fine if you take it later :)